index.js

```diff
@@ -1,10 +1,12 @@
-import { callRemoveBook, callAddBook, retrieveBookData } from './bookScript.js';
-import app from './singlePageApp.js';
+import { callRemoveBook, callAddBook, retrieveBookData } from './modules/bookScript.js';
+import app from './modules/singlePageApp.js';
+import getDateTime from './modules/DateTimeScript.js';
 
 const addBtn = document.querySelector('.add-btn');
 function onPageLoad() {
   retrieveBookData();
   app.init();
+  getDateTime();
 }
```

Comment:
- [ ] Kindly refactor this function to an arrow function as pointed out earlier.